### PR TITLE
qtrn 0.3 (new formula)

### PR DESCRIPTION
**Pull request by ackleymi**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by bfontaine** [comment]
The commit message should be "`qtrn 0.3 (new formula)`".

File: Formula/qtrn.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+    dir = buildpath/"src/github.com/FlashBoys/qtrn"
+    dir.install Dir["*"]
+    cd dir do
+      system "glide", "install"
+      system "go", "build", "-o", "build/qtrn",
+        "-ldflags",
+        "'-X=github.com/FlashBoys/qtrn/version.Version=#{version}'"
+      bin.install "build/qtrn"
```
Comment (bfontaine) [inline Formula/qtrn.rb]: Use `system "go", "build", "-o", bin/"qtrn", ...` to avoid this line.

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: Formula/qtrn.rb (hunk 1 above)
Comment (ackleymi) [inline Formula/qtrn.rb]: @bfontaine is there any way to avoid the repo popularity check from brew audit? Thats the failing check.

Comment (bfontaine) [inline Formula/qtrn.rb]: No, there’s no way. Ultimately it’s the maintainers’ decision to accept or reject new formulae but `brew audit` automates the basic checks. In your case this is a self-submission of a relatively new and not well-known project (3 stars on GitHub) so it doesn’t meet the [requirements](https://docs.brew.sh/Acceptable-Formulae.html#niche-or-self-submitted-stuff) to get in Homebrew core. You can, however, maintain a formula [in your own tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap.html) and re-submit it here once it gets more popular.

**Comment by fxcoudert** [comment]
Closing as per @bfontaine's comment.

**Pull request closed without merging**